app/code/Magento/Backend/Block/Menu.php

```diff
@@ -92,6 +99,7 @@ public function __construct(
         \Magento\Backend\Model\Auth\Session $authSession,
         \Magento\Backend\Model\Menu\Config $menuConfig,
         \Magento\Framework\Locale\ResolverInterface $localeResolver,
+        \Magento\Framework\App\Route\ConfigInterface $routeConfig,
```

Comment:
Please, inject new dependencies according to our [Backward compatible development guide](https://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/)